### PR TITLE
improving doc hpc

### PR DESCRIPTION
**Pull request by raphaeldussin**
following recommendations from the dev meeting, added more details and images

### CODE REVIEW

File: doc/large_problems_on_HPC.rst [hunk 1]
```diff
@@ -9,10 +17,19 @@ there are solutions to solve large regridding problems, provided you have access
 Performance Computing machine. Your ESMF installation (from conda or equivalent) comes with
 command line tools (ESMF_RegridWeightGen and ESMF_Regrid) that can be executed in parallel with
```
Comment (andersy005) [inline doc/large_problems_on_HPC.rst:18]: Could we add a link to the documentation page?

```suggestion
command line tools (`ESMF_RegridWeightGen and ESMF_Regrid <http://www.earthsystemmodeling.org/esmf_releases/public/ESMF_8_0_0/ESMF_refdoc/node3.html>`_) that can be executed in parallel with
```

**Review by andersy005 — APPROVED** [review]
Looks good to me 👍

**Comment by huard** [comment]
@raphaeldussin I suggest to fix the aspect ratio on the second figure to avoid stretching the map. 
https://pangeo-xesmf--22.org.readthedocs.build/en/22/large_problems_on_HPC.html

**Comment by raphaeldussin** [comment]
@huard better this way?

**Comment by huard** [comment]
Nope ; )
What I meant is that in the top figure, 1 horizontal degree takes the same space as one vertical degree. In the bottom figure, the vertical and horizontal ratio is not one, so the image appears stretched. It's not a big deal, but it distracts from the content. 
See http://xarray.pydata.org/en/stable/examples/visualization_gallery.html#Multiple-plots-and-map-projections

**Comment by raphaeldussin** [comment]
> Nope ; )
> What I meant is that in the top figure, 1 horizontal degree takes the same space as one vertical degree. In the bottom figure, the vertical and horizontal ratio is not one, so the image appears stretched. It's not a big deal, but it distracts from the content.
> See http://xarray.pydata.org/en/stable/examples/visualization_gallery.html#Multiple-plots-and-map-projections

oh I see what you mean, but the top figure is in meters in a polar Stereographic coordinate reference system and the bottom is in degrees of longitude/latitude. The latitudinal extent is relatively small compared to the longitudinal extent so it looks like a pancake if I set the aspect ratio to one :)

**Comment by raphaeldussin** [comment]
but more seriously, I don't want to bloat the repo with multiple versions of the figures, @huard do you approve the PR?

**Comment by huard** [comment]
@raphaeldussin Yes.

**Pull request merged**